crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -341,3 +336,71 @@ fn show_video_blob_info(
         }
     }
 }
+
+fn decoded_frame_ui(
+    ui: &mut egui::Ui,
+    frame_info: &FrameInfo,
+    timescale: re_video::Timescale,
+    source_image_format: &SourceImageDataFormat,
+) {
+    re_ui::list_item::list_item_scope(ui, "decoded_frame_ui", |ui| {
+        let default_open = false;
+        ui.list_item_collapsible_noninteractive_label("Decoded frame info", default_open, |ui| {
+            frame_info_ui(ui, frame_info, timescale);
+            source_image_data_format_ui(ui, source_image_format);
+        });
+    });
+}
+
+fn frame_info_ui(ui: &mut egui::Ui, frame_info: &FrameInfo, timescale: re_video::Timescale) {
+    let time_range = frame_info.time_range();
+    ui.list_item_flat_noninteractive(PropertyContent::new("Time range").value_text(format!(
+        "{} - {}",
+        re_format::format_timestamp_seconds(time_range.start.into_secs(timescale),),
+        re_format::format_timestamp_seconds(time_range.end.into_secs(timescale),),
+    )))
+    .on_hover_text("Time range in which this frame is valid.");
+
+    ui.list_item_flat_noninteractive(
+        PropertyContent::new("PTS").value_text(format!("{}", frame_info.presentation_timestamp.0)),
+    )
+    .on_hover_text("Raw presentation timestamp prior to applying the timescale.
+This specifies the time at which the frame should be shown relative to the start of a video stream.");
```

Comment:
For more even indentation:

```suggestion
    .on_hover_text("Raw presentation timestamp prior to applying the timescale.\n\
                    This specifies the time at which the frame should be shown relative to the start of a video stream.");
```